app-backend/batch/src/main/scala/stacExport/WriteStacCatalog.scala

```diff
@@ -60,6 +60,26 @@ final case class WriteStacCatalog(exportId: UUID)(
 ) extends Config
     with RollbarNotifier {
 
+  implicit class stacCollectionWithoutSelf(s: StacCollection) {
```

Comment:
This was the best way I could come up with to avoid a bunch of code duplication for filtering these links.

Argument could be made to add this to the actual classes themselves in gt-server but also seems like it should be a concern of the user to figure out what links to include/exclude